sparkly/utils.py

```diff
@@ -93,6 +181,45 @@ def kafka_get_topics_offsets(host, topic, port=9092):
     return offsets
 
 
+class lru_cache(object):
```

Comment:
I think it's a valid use case for the event that you are maintaining dataframes programatically / dynamically, rather than having them explicitly defined individually in code.